tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -17,35 +20,31 @@ class TestAddCapacity(ManageTest):
     """
     Automates adding variable capacity to the cluster while IOs running
     """
+
     @pytest.mark.parametrize(
         argnames=[
-            "node_multiplier", "capacity"
+            "capacity", "expected"
         ],
-        argvalues=[
-            pytest.param(
-                *[3, '2000Gi'],
-            ),
+        argvalues=[pytest.param(*['2000Gi', False]),
+                   pytest.param(*['6144Gi', False]),
+                   pytest.param(*['2048Gi', True]),
         ]
     )
-    def test_add_capacity(self, node_multiplier, capacity):
+    def test_add_capacity(self, capacity, expected):
         """
         Test to add variable capacity to the OSD cluster while IOs running
 
         Args:
-        node_multiplier: the number of OSD to add per worker node
-        capacity: the storage capacity of each OSD
+            capacity (String):the storage capacity of each OSD
+            expected (Boolean): Expected boolean statment from add_capacity function
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
```

Comment:
I don't think it matters to check for ipi and upi here. Capacity expansion is same for both. Node expansion differs.

Reply:
Done